src/lib.rs

```diff
@@ -179,10 +179,10 @@ mod blocking_reader;
 mod catalog;
 // Public as a workaround for: https://github.com/amazon-ion/ion-rust/issues/484
 pub mod constants;
-pub mod ion_eq;
 mod raw_symbol_token;
 mod raw_symbol_token_ref;
 // Public as a workaround for: https://github.com/amazon-ion/ion-rust/issues/484
+mod ion_data;
 pub mod reader;
```

Comment:
I think this comment is not aligned properly (`// Public as ...`)

Reply:
Good catch.